src/general/OptionsMenu.cs

```diff
@@ -516,17 +516,42 @@ private void OnDefaultsPressed()
         defaultsConfirmationBox.PopupCenteredMinsize();
     }
 
-    private void BackConfirmSelected()
+    private void BackSaveSelected()
+    {
+        // Save the new settings to the config file.
+        if (!Settings.Save())
+        {
+            GD.PrintErr("Failed to save new options menu settings.");
```

Comment:
It would be nice to show this error in a popup box so the user isn't confused in the unlikely scenario that the saving fails.